ChiModules/LinearBoltzmannSolver/IterativeOperations/lbs_setsource.cc

```diff
@@ -119,28 +120,28 @@ void LinearBoltzmann::Solver::
           q_mom[g] += inscat_g;
 
           //====================== Apply accross-groupset fission
-          if ((ell == 0) and (apply_fission_src))
+          if ( (ell == 0) and (apply_fission_src) )
```

Comment:
in this routine, why isn't I do not see anything related to delayed neutrons?

Reply:
Good question. Let me look at how easy it would be to implement otherwise I will create an issue.